README.md

```diff
@@ -813,4 +815,4 @@ The Circus Train logo uses the [Ewert font](http://www.1001fonts.com/ewert-font.
 # Legal
 This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).
 
-Copyright 2016-2019 Expedia Inc.
+Copyright 2016-2019 Expedia Group
```

Comment:
👍

Reply:
Not sure if you want to make that same change in all the files in that case ?

Reply:
Oh no, this is in all the licence headers, right? 😰

Reply:
Yeah. I guess, just leave it as is for now to keep it consistent with other BDP projects.

Reply:
But this being in the README is fine?

Reply:
I haven't received any guidance on whether this should remain `Inc.` or be changed to `Group` since the name change so I was just leaving everything as `Inc.` for now. I'll ask and let you all know.

Reply:
Ok, I changed it back to `Expedia Inc.` 🙂